com.unity.uiwidgets/Runtime/engine2/UIWidgetsPanelWrapper.cs

```diff
@@ -169,18 +172,30 @@ public partial class UIWidgetsPanelWrapper {
     }
 
     public void Initiate(IUIWidgetsWindow host, int width, int height, float dpr,
-        Dictionary<string, object> settings) {
+        Dictionary<string, TextFont> settings) {
+        
         D.assert(_renderTexture == null);
         _recreateRenderTexture(width, height, dpr);
 
         _handle = GCHandle.Alloc(this);
         _ptr = UIWidgetsPanel_constructor((IntPtr) _handle, (int) host.getWindowType(), UIWidgetsPanel_entrypoint);
         _host = host;
 
-       
-        _enableUIWidgetsPanel(JSONMessageCodec.instance.toJson(settings));
+        TextFont[] textFonts = loadTextFont(settings);
```

Comment:
we don't need to call loadTextFont.

we can change the function signature of fontsToObject to:
object fontsToObject(Dictionary<string, Font> fontSettings) and change the implementation a bit